dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/PLL/PLL.mo

```diff
@@ -17,22 +17,23 @@ model PLL "Phase-Locked Loop"
   import Dynawo.Types;
```

Comment:
General comments for this model. 
add.y is basically upu.im*cosphi - uPu.re*sinphi so you can replace all of this with a real expression.
Maybe gain + limInt could be replaced with a limited PI (to be checked)

Reply:
LimitedPI is not equivalent (the location of the limiter is different)